tests/math/test_matrix_manipulation.py

```diff
@@ -1234,7 +1258,9 @@ def test_valid_positive_definite(self, size):
         """Test that valid real, positive definite matrices work correctly"""
         # Create a positive definite matrix
         A = np.random.random((size, size))
-        mat = csr_matrix(np.eye(size) - 0.1 * (A @ A.T))
+        mat = np.eye(size) - 0.1 * (A @ A.T)
+        mat = _enforce_positivity(mat)
+        mat = csr_matrix(mat)
```

Comment:
Any concerns with this? Do we need to consider seeding the RNG?

Reply:
This test should pass any positive Hermitian matrices. We can definitely also seed it. I'll add in the next commit